app/services/stream-info.ts

```diff
@@ -5,7 +5,7 @@ import { Inject } from 'util/injector';
 import { StreamingService } from './streaming';
 import { HostsService } from 'services/hosts';
 import { authorizedHeaders } from 'util/requests';
-import { Subject } from 'rxjs';
+import { BehaviorSubject, Subject } from 'rxjs';
```

Comment:
What's the reason for using `BehaviorSubject` doing this? Do you have issues with this event in the ChildWindow?

Reply:
You can read the actual `stream-info` from the `state`, don't see why this must be changed here

Reply:
Given it represents a changing value rather than an event, `BehaviorSubject` is probably the correct type to use here.  I assume he's doing this avoid having to separately query the initial value?

Reply:
Reading from the state did not work if the user has just logged in, we changed to BehaviorSubject to be able to get the last emitted value regardless of subscription time (this component would subscribe after emit).

Reply:
> Reading from the state did not work if the user has just logged in
Seems like the must actualize the state when the user has logged in. We have an event for it.

If we are going to use `BehaviorSubject` we must think about some name convention for it to distinguish `BehaviorSubject` events from regular events